src/index.js

```diff
@@ -1,24 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import * as serviceWorker from './serviceWorker';
+import { Provider } from 'react-redux';
 
-import App from './App';
 import store from './redux/reduxStore';
```

Comment:
Consider renaming "reduxStore" to just "store"

Reply:
/fxd